rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -340,6 +340,19 @@ public void downloadClearingReports(
         }
     }
 
+    @PreAuthorize("hasAuthority('WRITE')")
+    @RequestMapping(value = PROJECTS_URL + "/{id}", method = RequestMethod.PATCH)
+    public ResponseEntity<Resource<Project>> patchProject(
+            @PathVariable("id") String id,
+            @RequestBody Project updateProject) throws TException {
+        User user = restControllerHelper.getSw360UserFromAuthentication();
+        Project sw360Project = projectService.getProjectForUserById(id, user);
+        sw360Project = this.restControllerHelper.updateProject(sw360Project, updateProject);
+        projectService.updateProject(sw360Project, user);
```

Comment:
Again, I am pretty sure you just used the implementation for Release and Component as a blueprint. But the `projectService.updateProject()` returns a `RequestStatus` object which we might want to inspect to check if the update succeeded.

Reply:
Hi @imaykay ,
There is a RequestStatus check happening inside projectService.updateProject(sw360Project, user) ,where it checks for the success status.

Reply:
Ah damn, I only checked the diff here in github and thought this would already be the thrift call.

The differentiation of the possible status is still not good in the service, but this should not be part of this ticket.